apps/website/src/pages/profile.tsx

```diff
@@ -119,6 +194,114 @@ const ProfilePage = withAuth(({ auth }) => {
   );
 });
 
+type ProfileAccountDetailsProps = {
+  userName: string | null | undefined;
+  userEmail: string;
+  tempName: string;
+  isSaving: boolean;
+  nameError: string;
+  onFocus: () => void;
+  onNameChange: (name: string) => void;
+  onSave: () => void;
+  onCancel: () => void;
+};
+
+const ProfileAccountDetails: React.FC<ProfileAccountDetailsProps> = ({
+  userName,
+  userEmail,
+  tempName,
+  isSaving,
+  nameError,
+  onFocus,
+  onNameChange,
+  onSave,
+  onCancel,
+}) => {
+  const showButtons = tempName !== (userName || '');
+  return (
+    <>
+      <H3>Account details</H3>
+      <div className="flex flex-col gap-4 container-lined bg-white p-8 h-fit">
+        <div className="profile-name">
+          <div className="flex flex-col gap-2">
+            <P><span className="font-bold">Name:</span></P>
+            <div className="flex gap-2 items-center">
```

Comment:
when it comes to BEM, i recommend naming all elements. this consistency makes it easier to navigate the code, prompts developers to think critically about their HTML semantic structures, and provides ready-to-go references for selective testing or more complex CSS.

you can read more about BEM here: https://getbem.com/

it is my personal preference from my experience, as ive seen this scale very well. but feel free to open this up for discussion and come up with your own preferred standards as a new dev team!

```suggestion
    <div className="profile-account-details">
      <H3 className="profile-account-details__title">Account details</H3>
      <div className="profile-account-details__container flex flex-col gap-4 container-lined bg-white p-8 h-fit">
        <div className="profile-account-details__name-container flex flex-col gap-2">
            <P className="profile-account-details__name-label"><span className="font-bold">Name:</span></P>
            <div className="profile-account-details__name-input-container flex gap-2 items-center">
```

Reply:
I've modified the CSS to follow the BEM structure. 

This is my first time using BEM alongside inline Tailwind and React, and I'm still on the fence about it. I've never really felt the need for all that extra verbosity when simple comments like {/* Course details */} or {/* Sharing section */} have done the job just fine. That said, I recognize it's a common pattern throughout the codebase, so I'll roll with it for now!